DAC/src/ringbuffer.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * ringbuffer.c
+ *
+ *  Created on: Feb 6, 2018
+ *      Author: sharana
+ *      Description: Program to initiate a ring buffer.
+ */
+#include"ringbuffer.h"
+#include<inttypes.h>
+/*
+ * Here we have a method put characters into the buffer.
+ * @param buffer it is a structure type to access the correct ring buffer.
+ * @param element it is the character.
+ */
+void put(RingBuffer* buffer, char element){
+	//Check space
+	 if(hasSpace(&buffer)== 1)
+	    {	//adding element to the buffer
```

Comment:
Move indentation to the left. Curly brackets should be lined up with statement above.
```
if(hasSpace(&buffer)== 1)
{
```
NOT
```
if(hasSpace(&buffer)== 1)
    {
```

Also, do we want curly brackets to the side of statements or on a separate line? We should decide on one or the other.